core/trino-main/src/test/java/io/trino/server/security/TestResourceSecurity.java

```diff
@@ -562,8 +561,8 @@ public void testJwtAuthenticatorWithInvalidAudience()
                         .put("http-server.authentication.jwt.key-file", HMAC_KEY)
                         .put("http-server.authentication.jwt.required-audience", TRINO_AUDIENCE)
                         .buildOrThrow())
+                .setSystemAccessControls(List.of(TestSystemAccessControl.NO_IMPERSONATION))
```

Comment:
why do we need `List.of` everywhere if we are setting single access control anway. Is there a place when we actually need a list? Maybe at least add helper method to avoid boilerplate.

Reply:
will do!